src/services/webhook.js

```diff
@@ -298,14 +301,37 @@ class WebhookController {
             .catch(err => {
                 if (err) {
                     if (retryCount < this.retryCount) {
-                        console.error('[WebhookController] Error occurred, trying again:', err);
+                        console.error(`[WebhookController] Error occurred, trying again for ${url}`);
                         this.sendEvents(events, url, retryCount++);
                     } else {
-                        console.error('[WebhookController] Error occurred, max retry count reached:', err);
+                        console.error(`[WebhookController] Error occurred, max retry count reached for ${url}`);
                     }
                 }
             });
     }
+
+    checkOnline() {
+        this.online = [];
```

Comment:
@Mygod

Reply:
This would cause race conditions. Disapprove.